<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Edit</class>
 <widget class="QWidget" name="Edit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="sideBar" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QTreeWidget" name="projectTreeWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>170</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>170</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="headerHidden">
         <bool>true</bool>
        </property>
        <column>
         <property name="text">
          <string>1</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <widget class="Developer::RuleEdit" name="ruleEdit"/>
     <widget class="Developer::ProgramEdit" name="programEdit"/>
     <widget class="Developer::GraphEdit" name="graphEdit"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Developer::RuleEdit</class>
   <extends>QWidget</extends>
   <header>ruleedit.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Developer::ProgramEdit</class>
   <extends>QWidget</extends>
   <header>programedit.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Developer::GraphEdit</class>
   <extends>QWidget</extends>
   <header>graphedit.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>projectTreeWidget</sender>
   <signal>itemClicked(QTreeWidgetItem*,int)</signal>
   <receiver>Edit</receiver>
   <slot>fileClicked(QTreeWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>88</x>
     <y>453</y>
    </hint>
    <hint type="destinationlabel">
     <x>93</x>
     <y>801</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>fileClicked(QTreeWidgetItem*)</slot>
 </slots>
</ui>
